### DISCUSSION TITLE
Vote for the new OpenRiak Logo!

### DISCUSSION THREAD

**Discussion by Bob-The-Marauder** [OpenRiak/.github#9] [Polls]
Five logos have been provided. They are listed in no particular order as 1 - 5 underneath.
1. ![Screenshot 2024-11-08 at 11 35 49 AM](https://github.com/user-attachments/assets/5654d6b6-4225-48d4-a3b9-eeb387b1483d)
2. ![Image from iOS](https://github.com/user-attachments/assets/43bdaaa0-285e-4c0f-92bc-3c9668bf297b)
3. ![Screen Shot 2024-11-12 at 2 31 50 AM](https://github.com/user-attachments/assets/91d6220b-5fb9-47c5-9fd0-ed8fef831fe4)
4. ![image](https://github.com/user-attachments/assets/1e0b546d-49c2-47bb-b1a1-d09632092bf1)
5. ![30240248](https://github.com/user-attachments/assets/13c1cc36-5107-4928-b72e-9f7c23613ea8)

The deadline for voting is 12 noon UTC on Sunday 1st December 2024.

**Comment by pjaclark** [comment]
My opinion:
- Logo 1 without the text doesn't have any relation to Riak for me but Ilike the dual-colour name
- Logo 2 won't look good when small; it would work better without the background lines
- Logo 3 will scale well but needs the name added - combining with Logo 5 for when logo and text desired would work well.
- Logo 4's text will not be legible when small (solve by having "open" on left, "riak" on right?) and the image doesn't have any relation to Riak for me
- Logo 5 is a tad plain - combine with the colours from Logo 1 and the image from Logo 3 to make a combined brand logo?

That combo of 1, 3 and 5 would look something like:
![image](https://github.com/user-attachments/assets/1a67a2ac-2aa4-4f00-a5cf-dff329c47c7a)
or
![image](https://github.com/user-attachments/assets/c6c74afb-88c5-4e71-8494-5d072b0f2984)

**Reply by JMercerGit** [reply]
The top option, with Riak centered rather than to the right, would be a good combination honestly.

**Reply by adkron** [reply]
I like the bottom one here. The words to the right are nice. Also the ratios are almost the golden ratio. Makes a good header

**Reply by stephenwood24-nhs** [reply]
I also prefer either of these combinations to the original options

**Reply by Bob-The-Marauder** [reply]
Bear in mind that we are in contact with all the logo creators, so I'm sure they would be happy to make a few modifications to their designs once the winning design is chosen.

**Reply by tburghart** [reply]
I too like the bottom one, and the separability of the components.

**Reply by leonross-nhs** [reply]
+1 for the bottom one from me..

**Reply by sanmiguel** [reply]
I was thinking basically the same thing as you've suggested here. IMHO 3 is the best logo, with and without a name for context.
I think a great nod to the project history would be to use the same palette as the original ring diagram and ditch (or lessen) the gradient effect. The current palette immediately brings to mind hex.pm's logo

**Comment by tburghart** [comment]
A note on # 5 - the original is a scalable vector font, so size and color(s) are fully adjustable.

**Comment by hmmr** [comment]
My vote is 3.

Also, if text is to be used with it, we should choose a more suitable font. Of course nobody was ever fired for going with Helvetica, but there surely will be more suitable options to consider. I suggest this one: https://fonts.google.com/specimen/Outfit?icon.size=24&icon.color=%235f6368&categoryFilters=Sans+Serif:%2FSans%2FGeometric&preview.text=riak&preview.size=72, which I have 95% confidence was actually the font used by Basho designers.

**Comment by matthewvon** [comment]
Voting for number 1 because it is closest to one of the old logos for Riak vnode / hash circle.

https://lion.matthewv.org/download/riak-powered-light.png

https://lion.matthewv.org/download/riak-powered.svg

**Comment by tburghart** [comment]
For comparison purposes, this is the text logo rendered using the Outfit font (original was Verdana) with colors taken from # 3:

<img width="576" alt="OpenRiakColor" src="https://github.com/user-attachments/assets/82baad24-cc8d-4bfd-b3a9-cc35c9d46e07">

**Reply by tburghart** [reply]
For reference, it's a Pages document with some kerning and other tricks applied to get the desired character alignment.
Even though it _looks_ as though the 'p' and 'R' aren't perfectly aligned, I think that's due to display pixel characteristics - when both are black, the left line is sharp and straight.
The 'n' and 'k' verticals are _not_ perfectly aligned, which I'd correct by adding a smidge of space between the 'R' and 'i', but not bothering unless it's what we choose to use.
And yes, the color swap to 'Open' in orange and 'Riak' in blue is deliberate - I think it emphasizes Riak as the constant - but obviously everything about the colors is infinitely adjustable.

**Comment by Bob-The-Marauder** [comment]
The winner is Logo 3 by Robert Carbone! Congratulations, Robert!
Thank you to everyone who participated in the vote. Voting is now closed.

**Reply by tburghart** [reply]
I agree with # 3 being the best graphic, but I do think we should have a consistent pairing of the logo with "OpenRiak" text for consistency - which appears to have been the primary topic of discussion.

**Comment by intjelic** [comment]
I can't believe Logo 2 4 and 5 were even considered 😂.

Logo 3 is stunning tho ;)

**Reply by adkron** [reply]
These were somewhat rough ideas. Some were more fleshed out than others. Constructive criticism is always welcome, but make sure t is helpful.

**Reply by intjelic** [reply]
Here is my constructive criticism then ;) 

OpenRiak is an amazing initiative. However, it's scary that those ugly ass logos were even suggested. I don't think anyone would take this project seriously if logo 2, 4 and 5 were chosen.
